lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -354,7 +368,8 @@ follow nl tr cps yield header =
         Right (RollForward cursor' tip (blockFirst : blocksRest)) -> do
             let blocks = blockFirst :| blocksRest
             traceWith tr $ MsgApplyBlocks (header <$> blocks)
-            action <- yield blocks tip
+            params <- getTxParameters nl
```

Comment:
I wonder what's the cost of this. In practice, it should only happen every 1000 blocks in "restoration mode" so I assume it's a small cost. Yet, since we know it can only change with epoch changes, if it ends up making the wallet choke every 1000 blocks we may need something clever here :thinking:

Reply:
There is almost zero cost -- it's reading a TVar.